tests/e2e/default/tasks/tasks.wdio-spec.js

```diff
@@ -41,8 +130,68 @@ describe('Tasks', () => {
     await browser.refresh();
   });
 
-  afterEach(async () => {
-    await utils.revertSettings(true);
+  it('should load multiple pages of tasks on infinite scrolling', async () => {
+    const settings = await compileTasks('tasks-multiple-config.js');
+    await utils.updateSettings(settings, { ignoreReload: 'api', sync: true });
+
+    await commonPage.goToTasks();
+    const list = await tasksPage.getTasks();
+    const infos = await tasksPage.getTasksListInfos(list);
+    expect(infos).to.have.length(201);
+    for (let i = 0; i < (infos.length/3); i++) {
+      expect(infos).to.include.deep.members([
+        {
+          contactName: 'Mary Smith',
+          formTitle: `person_create_${i + 1}`,
+          lineage: '',
+          dueDateText: 'Due today',
+          overdue: true
+        },
+        {
+          contactName: 'patient1',
+          formTitle: `person_create_${i + 1}`,
+          lineage: 'clinic2',
+          dueDateText: 'Due today',
+          overdue: true
+        },
+        {
+          contactName: 'patient2',
+          formTitle: `person_create_${i + 1}`,
+          lineage: '',
+          dueDateText: 'Due today',
+          overdue: true
+        },
+      ]);
+    }
+    await tasksPage.scrollToLastTaskItem();
+    const loadingStatusSelector = await $('#tasks-list p.loading-status');
+    const elementText = await loadingStatusSelector.getText();
+    expect(elementText).to.contain('No more tasks');
```

Comment:
We try not to add selectors in the test files. Please create a separate method in `tasks.wdio.page.js` that will return the text for the loading status element.
And can you please try to create the selector using the text of the element, as suggested in the [wdio page](https://webdriver.io/docs/selectors/)? If you use the selector with the text the assert will be just to validate that the element is visible, like we are doing [here](https://github.com/medic/cht-core/blob/3e39f6f5e38fc4a01fe5ef0dc1e328907cb33d8a/tests/page-objects/default/enketo/common-enketo.wdio.page.js#L30-L32)